.circleci/config.yml

```diff
@@ -146,6 +180,9 @@ jobs:
       - checkout
       - attach_workspace:
           at: ./
+      - run:
+          name: Move build
```

Comment:
`Move Production build`

just so folks see the steps and know *for sure* what's happening